composer.json

```diff
@@ -8,17 +8,18 @@
   "require": {
     "php": "~7.2.0",
     "wp-cli/wp-cli": "~0.22",
-    "wpackagist-plugin/amazon-s3-and-cloudfront": "~1.4.2",
+    "wpackagist-plugin/amazon-s3-and-cloudfront": "~2.0",
     "wpackagist-plugin/amazon-web-services": "^1.0",
-    "wpackagist-plugin/insert-html-snippet": "~1.2.4",
-    "wpackagist-plugin/perfect-woocommerce-brands": "~1.6.5",
+    "wpackagist-plugin/insert-html-snippet": "~1.2.5",
+    "wpackagist-plugin/akismet": "~4.1.0",
```

Comment:
Move `akismet` to composer and remove the source code in wp-content/plugins/akismet from version control.